src/System.Diagnostics.Process/tests/ProcessTest_ConsoleApp/ProcessTest_ConsoleApp.cs

```diff
@@ -7,45 +8,40 @@ class Program
     {
         static int Main(string[] args)
         {
-            try
+            if (args.Length > 0)
             {
-                if (args.Length > 0)
+                if (args[0].Equals("infinite"))
                 {
-                    if (args[0].Equals("infinite"))
-                    {
-                        // To avoid potential issues with orphaned processes (say, the test exits before
-                        // exiting the process), we'll say "infinite" is actually 30 seconds.
-                        
-                        Console.WriteLine("ProcessTest_ConsoleApp.exe started with an endless loop");
-                        Thread.Sleep(30 * 1000);
-                    }
-                    if (args[0].Equals("error"))
-                    {
-                        Exception ex = new Exception("Intentional Exception thrown");
-                        throw ex;
-                    }
-                    if (args[0].Equals("input"))
-                    {
-                        string str = Console.ReadLine();
-                    }
+                    // To avoid potential issues with orphaned processes (say, the test exits before
+                    // exiting the process), we'll say "infinite" is actually 30 seconds.
+                    Thread.Sleep(30 * 1000);
```

Comment:
Understood, but there will be some things out of xunit's control.

Regardless, if you'd prefer to wait-and-see, ok.  My concern is just that on my own box, the process tests have taken upwards of 10 seconds, which in a lab environment could easily swing past 30 seconds, and now that there's a single infinite process for the duration of the tests, it's more likely that'll happen.